types/redis/index.d.ts

```diff
@@ -93,13 +94,15 @@ export interface OverloadedListCommand<T, U, R> {
 }
 
 export interface OverloadedSetCommand<T, U, R> {
-    (key: string, arg1: T, arg2: T, arg3: T, arg4: T, arg5: T, arg6: T, cb?: Callback<U>): R;
-    (key: string, arg1: T, arg2: T, arg3: T, arg4: T, arg5: T, cb?: Callback<U>): R;
-    (key: string, arg1: T, arg2: T, arg3: T, arg4: T, cb?: Callback<U>): R;
-    (key: string, arg1: T, arg2: T, arg3: T, cb?: Callback<U>): R;
-    (key: string, arg1: T, arg2: T, cb?: Callback<U>): R;
-    (key: string, arg1: T | { [key: string]: T } | T[], cb?: Callback<U>): R;
-    (key: string, ...args: Array<T | Callback<U>>): R;
+    (key: string | string[], arg1: T, arg2: T, arg3: T, arg4: T, arg5: T, arg6: T, cb?: Callback<U>): R;
```

Comment:
do you need this `string[]` anymore? it should be accounted for by the two new types i think

Reply:
Oops. I missed that. I'll fix it and push right away. Thank you.

Reply:
Updated. Kindly review. Thank you.